api/spec/services/issues_feed_builder_spec.rb

```diff
@@ -36,5 +38,19 @@
         expect(result.pluck(:comments_count)).to eq [10, 5]
       end
     end
+
+    context 'when filtering by categories' do
+      let(:category) { create(:issue_category) }
+      let(:filter_params) { { category: category.name } }
+
+      before do
+        create(:issue, issue_category: category, product:)
+        create(:issue, product:)
+      end
+
+      it 'returns issues filtered by categories' do
+        expect(result.pluck(:issue_category_id)).to eq [category.id]
+      end
+    end
```

Comment:
does 'context' the same of 'describe' ?

what about border case: `return issues if filter_params.blank? || filter_params[:category].blank?`

Reply:
> does 'context' the same of 'describe' ?

yeah, it is the same thing

about the edge case, I think it's being covered by the other context that tests if sorting is passed, because in that case filters are not being passed hahah